benchmark-graphs/src/main/resources/html/header.html

```diff
@@ -0,0 +1,7 @@
+<html>
+<head>
+  <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+  <script>
+var onload = function() {
+
+var plotlydata =
```

Comment:
Maybe I should make this into a pseudo-template with `{#data#}` inserted here instead? What do you think?